tests/lib/apt.sh

```diff
@@ -11,6 +11,9 @@ install_build_snapd(){
         apt install -y --only-upgrade snapd
         mv sources.list.back /etc/apt/sources.list
         apt update
+        if [ "$SPREAD_REBOOT" = 0 ]; then
```

Comment:
We use the `if` format in the whole suite, I personally find it much more readable. Additionally, it allows us to use echo to let know to anyone trying to debug the test what have you just checked (not in this case though, in which the thing being checked is clear enough).